lesson_19/page_objects/computer_product_page.py

```diff
@@ -0,0 +1,106 @@
+from selenium.webdriver.common.by import By
+
+from ..page_objects.base_page import BasePage
+from .cart_page import CartPage
+
+
+class ComputerProductPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    prices = []
+
+    __product_title = (By.XPATH, "//div[@class='product-name']//h1")
+    __cart_page = (By.XPATH, "//li[@id='topcartlink']//a[@href='/cart']")
+    __processor_radio_buttons = {
+        'slow': (By.XPATH, "//label[contains(text(), 'Slow')]"),
+        'medium': (By.XPATH, "//label[contains(text(), 'Medium')]"),
+        'fast': (By.XPATH, "//label[contains(text(), 'Fast')]")
+    }
+    __ram_radio_buttons = {
+        '8_gb': (By.XPATH, "//label[contains(text(), '8 GB') or contains(text(), '8GB')]"),
+        '2_gb': (By.XPATH, "//label[contains(text(), '2 GB') or contains(text(), '2GB')]"),
+        '4_gb': (By.XPATH, "//label[contains(text(), '4 GB') or contains(text(), '4GB')]")
+    }
+    __hdd_radio_buttons = {
+        '320_gb': (By.XPATH, "//label[contains(text(), '320 GB')]"),
+        '400_gb': (By.XPATH, "//label[contains(text(), '400 GB')]")
+    }
+    __software = {
+        'image_viewer': (By.XPATH, "//label[contains(text(), 'Image Viewer')]"),
+        'office_suite': (By.XPATH, "//label[contains(text(), 'Office Suite  [+100.00]')]"),
+        'other_office_suite': (By.XPATH, "//label[contains(text(), 'Other Office Suite  [+40.00]')]")
+    }
+    __price = (By.XPATH, "//span[@itemprop='price']")
+    __qty = (By.XPATH, "//input[@class='qty-input']")
+    __add_to_cart = (By.XPATH, "//div[@class='add-to-cart']//input[@type='button']")
+
+    def set_processor_radio(self, value: str, get_text: bool = False):
```

Comment:
why get_text is bool value? bool variables names should starts with is_

Reply:
I decided to combine this method. It can set some radio or it can return a text of this radio. It is important because text contains a cost of additional options. 
Naming for variable is fixed